### ISSUE TITLE
Meta+shift+backspace not working

### ISSUE THREAD

**Issue by AckslD** [david-janssen/kmonad#193]
I just got started with kmonad and l'm loving it! I use the following alias for space
```
sme (tap-hold-next-release 500 space lmet)
```
to use it as meta (windows key) when held which I use a lot in my wm. It all works great except one key combination I have which is when I hold space (now meta) + shift + backspace. Other similar key combinations such as space+shift+c do work.

Any idea why this might be the case?

**Comment by thoelze1** [comment]
Could be a rollover issue? That is, maybe your keyboard doesn't support space and backspace being pressed simultaneously. When I first bound my modifiers to my home row and spacebar I discovered that a few combinations didn't work.

**Comment by AckslD** [comment]
Thanks for your reply @thoelze1! I see, I guess that's not possible to fix if it's dictated by the keyboard. Did you find any solution or did you just avoid those key combinations?

**Comment by thoelze1** [comment]
Did you determine that it is indeed the hardware being unable to emit those keys simultaneously?

Yeah, because it's a hardware issue there's definitely no simple fix. I ended up avoiding those combos. For me it was one of the brackets causing problems. I looked at it as an opportunity to improve the combos. IMO it makes no sense that important keys like shift and backspace are in the corners of the ANSI keyboard
